### DISCUSSION TITLE
[E971] Found incompatible lengths in `Doc.from_array`

### DISCUSSION THREAD

**Discussion by KangChou** [explosion/spaCy#10810] [Help: Coding & Implementations]
```
File "spacy/tokens/doc.pyx", line 1033, in spacy.tokens.doc.Doc.from_array ValueError: [E971] Found incompatible lengths in `Doc.from_array`: 6 for the array and 13 for the Doc itself.
```

## Your Environment

* Operating System:ubuntu16.04
* Python Version Used:3.6.13
* spaCy Version Used:3.1
* Environment Information:anaconda3+ubuntu+python3.6

**Comment by polm** [comment]
It sounds like you got the error you shared when running some code with spaCy. With just the error message we can't help you.

Can you give a short sample of code that gives the error? Then maybe we can help you find what went wrong.
